app/database/queries/breeze_buddy/analytics.py

```
@@ -70,9 +88,25 @@ def build_analytics_where_clause(
         conditions.append(f"lct.call_initiated_time < ${len(values) + value_offset}")
 
     # Standard column filters
+    # Template filter - supports BOTH template name and template_id for backward compatibility
     if "template" in filters and filters["template"]:
```

Ruff (0.14.10)

Unnecessary key check before dictionary access

Replace with `dict.get`

(RUF019)